.prettierignore

```diff
@@ -2,6 +2,8 @@
 bellman-cuda
 sdk/zksync-rs/CHANGELOG.md
 CHANGELOG.md
+# prettier breaks [!NOTE] formatting
```

Comment:
That's a bummer. Isn't it possible to disable prettier for a specific line / fragment using comments? Would be useful in this case in order to not switch off formatting for the entire files.

Reply:
I found that you can use
`<!-- prettier-ignore-start -->`
and
`<!-- prettier-ignore-end -->`